### DISCUSSION TITLE
why not use --seed replace --fold ?

### DISCUSSION THREAD

**Discussion by Silentdoer** [nushell/nushell#7400] [Q&A]
[ 1 2 3 4 ] | reduce --fold 10 {|it, acc| $acc + $it }

**Comment by kubouch** [comment]
Please, when asking something, make a descriptive title and formulate your question clearly. You've recently filed a large number of low-effort issues and questions that just waste everybody's energy trying to figure out what you meant.

The `--fold` comes from iterator terminology used in Rust, I believe, if that's what you asked about.

**Reply by Silentdoer** [reply]
i am sorry, i just think replace --fold with --seed or --init will more clear
